lib/faerie/src/backend.rs

```diff
@@ -59,16 +70,36 @@ impl FaerieBuilder {
             format,
             faerie_target,
             collect_traps,
+            libcall_names: Box::new(libcall_names),
+        })
+    }
+
+    /// Default names for `ir::LibCall`s. A function by this name is imported into the object as
+    /// part of the translation of a `ir::ExternalName::LibCall` variant. Calls to a LibCall should
+    /// only be inserted into the IR by the `cretonne_codegen` legalizer pass.
+    pub fn default_libcall_names() -> Box<Fn(ir::LibCall) -> String> {
+        Box::new(move |libcall| match libcall {
+            ir::LibCall::Probestack => "__cretonne_probestack".to_owned(),
+            ir::LibCall::CeilF32 => "ceilf".to_owned(),
+            ir::LibCall::CeilF64 => "ceil".to_owned(),
+            ir::LibCall::FloorF32 => "floorf".to_owned(),
+            ir::LibCall::FloorF64 => "floor".to_owned(),
+            ir::LibCall::TruncF32 => "truncf".to_owned(),
+            ir::LibCall::TruncF64 => "trunc".to_owned(),
+            ir::LibCall::NearestF32 => "nearbyint".to_owned(),
+            ir::LibCall::NearestF64 => "nearbyintf".to_owned(),
```

Comment:
Thanks for making this change. As we get more experience with this, it may want to evolve, but this looks good for now.